Update Azure Database Explorer Output with streaming ingestion recommendation (#111226)
Azure/azure-content: articles/stream-analytics/azure-database-explorer-output.md

* The name of the columns and data type should match between the Azure Stream Analytics SQL query and the Azure Data Explorer table. The comparison is case-sensitive.
* Columns that exist in your Azure Data Explorer clusters but are missing in Azure Stream Analytics are ignored. Columns that are missing in Azure Stream Analytics raise an error.
* The order of your columns in your Azure Stream Analytics query doesn't matter. The schema of the Azure Data Explorer table determines the order.

> * Azure Data Explorer has an aggregation (batching) policy for data ingestion that's designed to optimize the ingestion process. The policy is configured to 5 minutes, 1,000 items, or 1 GB of data by default, so you might experience a latency. To reduce latency, enable streaming ingestion on your cluster and then table or database using these [steps](/azure/data-explorer/ingest-data/data-explorer-ingest-data-streaming). For aggregation options, see [IngestionBatching policy](/azure/data-explorer/kusto/management/batchingpolicy).

Reviewer: ```suggestion
* Azure Data Explorer has an aggregation (batching) policy for data ingestion that's designed to optimize the ingestion process. The policy is configured to 5 minutes, 1,000 items, or 1 GB of data by default, so you might experience a latency. To reduce latency, enable streaming ingestion on your cluster, and then table or database by following the steps in [Configure streaming ingestion on your Azure Data Explorer cluster](/azure/data-explorer/ingest-data-streaming). For aggregation options, see [IngestionBatching policy](/azure/data-explorer/kusto/management/batchingpolicy).
```
the link went to a 404. i've corrected it.
